jquery.js

```diff
@@ -0,0 +1,94 @@
+var scores = new Scores();
+$( document ).ready(function(){
+  var game = new Game();
+  var frame = new Frame();
+
+  $('#right').hide();
+
+  $('#new_game').click(function(){
+    $('#right').hide();
+    game.newGame(frame)
+    $('.score1').empty();
+    $('.score2').empty();
+    $('.score3').empty();
+    $('.total').html('')
+    $('.scorecard p').html('')
+  })
+
+  $('#save').click(function(event){
+    event.preventDefault();
+    game.get_score(frame)
+    scores.save_score(($("#name").val()),game.score)
+    $('#scores').append("<br>" + scores.scores.slice(-1)[0][0] + "  ---->  " + scores.scores.slice(-1)[0][1])
+  })
+
+  $('#bowl').click(function(){
```

Comment:
This function has too many statements. (55)<br>This function's cyclomatic complexity is too high. (11)